location_register/converter/koatuu.py

```diff
@@ -1,148 +1,238 @@
+        table_model_objects = table_model.objects.all() # table_model is a model from ratu_models
         for table_record in table_model_objects: # dictionary for keeping whole model class objects
-            if (table_record.name):
+            if table_record.name:
                 unit_name = table_record.name + str(table_record.region_id)
-                koatuu_dict[unit_name]=table_record
+                koatuu_dict[unit_name] = table_record
         return koatuu_dict
 
-    #creating a dictionary out off the values, which already exists in citydistrict table
-    def create_dictionary_for_citydistrict_table(self):#table_medel is one model from ratu_models
+    # creating a dictionary out off the values, which already exists in citydistrict table
+    def create_dictionary_for_citydistrict_table(self):# table_model is a model from ratu_models
         koatuu_dict = {}
         table_model_objects = Citydistrict.objects.all()
         for table_record in table_model_objects: # dictionary for keeping whole model class objects
-            if (table_record.name):
+            if table_record.name:
                 unit_name = table_record.name + str(table_record.city_id)
-                koatuu_dict[unit_name]=table_record
+                koatuu_dict[unit_name] = table_record
         return koatuu_dict
-    
 
-    #creating list out of name and district_id from json file for district items
-    def create_district_items_list(self, object_koatuu, object_region_name, region_dict, object_district_name, KOATUU_VALUE):
+    # creating list out of name and district_id from json file for district items
+    def create_district_items_list(self, object_region_name, region_dict, \
+        object_district_name, koatuu_value):
         district_items_list = []
-        if not (object_region_name in region_dict):
+        if not object_region_name in region_dict:
             return
-        region_koatuu = region_dict[object_region_name]            
-        if not (KOATUU_VALUE[:2] == str(region_koatuu.koatuu)[:2]):
+        region_koatuu = region_dict[object_region_name]
+        if not koatuu_value[:2] == str(region_koatuu.koatuu)[:2]:
             return
         object_district_name = object_district_name + str(region_koatuu.id)
         district_items_list.append(object_district_name)
         return district_items_list
 
-    #creating list out of name and district_id from json file fot city items
-    def create_city_items_list(self, data, object_koatuu, city_dict, district_items_list, KOATUU_VALUE):
-        city_list =[]
+    # creating list out of name and district_id from json file fot city items
+    def create_city_items_list(self, city_dict, district_items_list, koatuu_value):
+        city_list = []
         for object_district_name in district_items_list:
-            if (object_district_name in city_dict) & (KOATUU_VALUE[2] =='1'):
+            if (object_district_name in city_dict) & (koatuu_value[2] == '1'):
```

Comment:
what? & operator?? wrong way...
 ```suggestion
            if object_district_name in city_dict and koatuu_value[2] == '1':
```